measurements/api/legacy_explorer.py

```diff
@@ -0,0 +1,135 @@
+'''
+prefix: /api/legacy_explorer
+
+That's legacy API for old version of OONI Explorer that mimics (and replaces)
+API at http://explorer.ooni.torproject.org/api/. That's a hack, don't use it.
+This API will eventually be refactored & split across private.py and public API.
+'''
+
+import json
+import itertools
+
+from flask import Blueprint, current_app, request
+from flask.json import jsonify
+from sqlalchemy import func # , or_, and_, false, true
+
+from measurements.models import Report, Input, Measurement, Autoclaved
+from measurements.api.measurements import fetch_autoclaved_text
+
+# prefix: /api/legacy_explorer
+api_legacy_explorer_blueprint = Blueprint('api_legacy_explorer', 'measurements')
+
+## $ <access_log grep 'GET.*HTTP/' | sed 's,.*GET ,,; s,[? ].*,,' | grep ^/api | sort | uniq -c
+##      1 /api/countries
+##    757 /api/countries/findOne
+##    992 /api/nettests
+##  13829 /api/nettests/findOne
+##  88191 /api/reports
+##  18663 /api/reports/asnName
+##   1065 /api/reports/blockpageCount
+##     35 /api/reports/blockpageDetected
+##   1760 /api/reports/blockpageList
+##   1111 /api/reports/count
+##    973 /api/reports/countByCountry
+##     26 /api/reports/total
+##   2899 /api/reports/vendors
+##   2071 /api/reports/websiteDetails
+##   2446 /api/reports/websiteMeasurements
+
+## Something that has data:
+## https://explorer.ooni.torproject.org/api/reports/vendors?probe_cc=DE
+
+GOOD_REPORTS_ORDER = set('{} {}'.format(*t) for t in itertools.product(
+        ('input', 'probe_asn', 'probe_cc', 'test_name', 'test_start_time'),
+        ('ASC', 'DESC')))
+
+## {"test_start_time":{"between":["",""]}}
+## {"test_start_time":{"between":["",""]}}
+@api_legacy_explorer_blueprint.route('/reports')
```

Comment:
Everything that is requested via the `/reports$` endpoints is already supported almost entirely (except for the link to the measurement) inside of this branch: https://github.com/TheTorProject/ooni-explorer/pull/119 that requires only ooni-explorer changes and works by querying the public endpoints.

I would suggest to maybe focus on implementing the the other endpoints (no need to have them be exactly 1-1 compatible, but as long as the needed data is exposed I will wire them up to explorer).

The wiring up to explorer is really not a lot of work, if you consider that to replace the `/reports` endpoint with the new measurements API it took about 2h of work. Now that I have figured out how to do it, doing the rest is going to be pretty simple.